frog/imports/ui/GraphEditor/SidePanel/fileConvert.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React, { Component } from 'react';
+import Form from 'react-jsonschema-form';
+import CopyToClipboard from 'react-copy-to-clipboard';
+
+import { Uploads } from '/imports/api/uploads';
+
+class fileConvert extends Component {
+  state: { url: string, copied: boolean, display: boolean };
+
+  constructor(props: {}) {
+    super(props);
+    this.state = { url: '', copied: false, display: false };
+  }
+
+  onChange = (e: { formData: { file: string } }) => {
+    const that = this;
+    Uploads.insert(e.formData.file, (err, fileObj) => {
+      that.setState({
```

Comment:
If I switch tab or close the sidepanel too fast, I receive the following error
```
warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op. Please check the code for the fileConvert component.
```

To fix that you should use the methods `componentDidMount` and `componentWillUnmount` and for example maintain a variable `this.mounted` and test `if(mounted) { setState(...) }`

https://facebook.github.io/react/docs/react-component.html